aries-site/src/examples/templates/pagination/PaginationListExample.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Box, List, Menu } from 'grommet';
+import { More } from 'grommet-icons';
+
+const data = [];
+
+for (let i = 0; i < 95; i += 1) {
+  data.push({
+    entry: `entry-${i + 1}`,
+  });
+}
+
+export const PaginationListExample = () => (
+  <Box overflow="auto" pad="xsmall">
+    <List
+      data={data}
+      action={(item, index) => (
+        <Menu
+          key={index}
+          icon={<More />}
+          hoverIndicator
+          items={[{ label: 'one' }]}
```

Comment:
Not sure if it matters that each label is 'one' could also use a for loop like above

Reply:
Maybe we should change this to some action , cause this is the drop that appears if you click the "..." on the list item. So maybe something like "See Details" or something